src/app/shared/filter-bar/filter-bar.component.ts

```diff
@@ -80,13 +80,29 @@ export class FilterBarComponent implements OnInit, AfterViewInit, OnDestroy {
     this.views$?.value?.forEach((v) => (v.retrieveFilterable().filter = filterValue));
   }
 
+  /**
+   * Changes status to a valid, default value when an incompatible combination of type and status is encountered.
+   */
+  checkValidCombination() {
+    if (this.dropdownFilter.status === 'merged' && this.dropdownFilter.type === 'issue') {
+      this.dropdownFilter.status = 'all';
+    }
+  }
+
```

Comment:
Could we add some logic to change the dropdown filter type to Pull Request if Merged is selected? 

I don't think we need this logic? Because it seems impossible to select Merged when Issue is selected

Reply:
I've changed it so that PR will be selected if merged is selected!

I think we will still need to check for the combination 'issue + merged' because users can select 'pr + merged' and then change the type from pr to issue. 

Another possibility could be to remove the 'issue' option when 'merged' is selected, but this would mean more coupling and I'm not sure if this would cause the behaviour to deviate too much from the original.

Reply:
Better to not hide the Issue option, since that filter is more important than filtering by status. Thanks this implementation is good